src/Control/Monad/Codensity.hs

```diff
@@ -36,6 +36,8 @@ module Control.Monad.Codensity
   , codensityToComposedRep, composedRepToCodensity
   , wrapCodensity
   , improve
+  , reset
```

Comment:
Perhaps include a

```
-- ** Delimited continuations
```

Haddock section header above this line for symmetry with `Control.Monad.Trans.Cont`, which does something similar. The Haddocks for `shift`/`reset` draw upon the terminology of delimited continuations, after all, which sets them apart from other functions in this module, which describe things purely in terms of category theory. That's fine, but it's worth having a distinct section for these.